src/org/daisy/dotify/formatter/impl/search/VolumeData.java

```diff
@@ -0,0 +1,60 @@
+package org.daisy.dotify.formatter.impl.search;
+
+/**
+ * <p>Provides basic volume data for identifiers.</p>
```

Comment:
Can you provide some more documentation for this class? As a first time reader who hasn't looked at the other code yet I don't quite get what it is supposed to do.

Reply:
OK I think I get it, but still a bit more explanation wouldn't hurt, otherwise we have to organize another two-days documentation spree in a couple of years when a new developer joins the team.

Reply:
I would welcome a new developer for Dotify! ;-)
Done.

Reply:
Great! Maybe also add a Javadoc for `getVolumeNumber` and `isAtStartOfVolumeContents`? The method names are already pretty self-explaining though.